receiver/chronyreceiver/README.md

```diff
@@ -0,0 +1,64 @@
+# Chrony Receiver
+
+| Status                   |           |
+| ------------------------ | --------- |
+| Stability                | [alpha]   |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
+
+The [chrony] receiver is a pure go implementation of the command `chronyc tracking` to allow for
+portability across systems and platforms. All of the data that would typically be captured by
+the tracking command is made available in this receiver, see [documentation](./documentation.md) for
+more details.
+
+## Configuration
+
+### Default
+
+By default, the `chrony` receiver will default to the following configuration:
+
+```yaml
+chrony/defaults:
+  address: unix:///var/run/chrony/chronyd.sock # The default port by chronyd to allow cmd access
+  timeout: 10s # Allowing at least 10s for chronyd to respond before giving up
+
+chrony:
+  # This will result in the same configuration as above
+```
+
+### Customised
+
+The following options can be customised:
+
+- address (required) - the address on where to communicate to `chronyd`
+  - The allowed formats are the following
+    - udp://hostname:port
+    - unix:///path/to/chrony.sock (Please note the triple slash)
+- timeout (optional) - The total amount of time allowed to read and process the data from chronyd
+  - The value of the timeout must be greater than or equal to 1s
+- collection_interval (optional) - how frequent should this receiver should poll [chrony]
+- metrics (optional) - what metrics should be exported, read the [documentation] for complete details
```

Comment:
As a non-native speaker, I feel like there's a lesson for me to learn here :-) Why did you pick "what" instead of "which"?

Reply:
This is me being lazy with English :disappears: